src/Sparrow/Json/PerCoreContainer.cs

```diff
@@ -10,11 +10,16 @@ namespace Sparrow.Json
     internal sealed class PerCoreContainer<T> : IEnumerable<(T Item, (int, int) Pos)>
         where T : class
     {
+        private readonly int _numberOfSlotsPerCore;
         private readonly T[][] _perCoreArrays;
+        private readonly int[] _perCoreArrayLength;
```

Comment:
We should probably change that so we have at least each `length` 64 bytes from the next one.
Right now likely have a lot of false sharing here.

Reply:
done